src/components/organisms/UpComingAlert.tsx

```diff
@@ -32,6 +31,12 @@ const UpComingAlert = () => {
         description="Lorem Ipsum"
         time={new Date()}
       />
+      <EmergencyAlertCard
+        sender="Lorem Ipsum"
+        title="Lorem Ipsum"
+        description="Lorem Ipsum"
+        time={new Date()}
+      />
```

Comment:
Is this connected to backend yet?

Reply:
This is not connected yet, but it will be implemented along with other notifications.

Reply:
I see